app/controllers/users_controller.rb

```diff
@@ -1,4 +1,27 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+    #debugger
+  end
+
   def new
+    @user = User.new
+  end
+
+  def create
+  @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to user_url(@user)
+    else
+      render 'new'
+    end
   end
+
+  private
+
```

Comment:
I prefer this style  
- no blank line after `private` (optional)
- align `def user_params` as same column with `private` (mandatory)